groups/cvo2/security_group_rules.tf

```diff
@@ -51,7 +51,7 @@ resource "aws_security_group_rule" "onpremise_admin" {
 resource "aws_security_group_rule" "mediator_ssh" {
   count = var.create_security_group ? 1 : 0
   
-  security_group_id = local.sg_id["security_group_id"]
+  security_group_id = data.aws_security_group.mediator.id
```

Comment:
How is this being applied? Form what I can see this security group is in `shared-services` but this codebase is for the heritage accounts and I don't see any cross-account config.